pages/common/pio-check.md

```diff
@@ -0,0 +1,20 @@
+# pio check
+
+> Perform a static analysis check on a PlatformIO project.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_check.html>.
+
+- Perform a basic analysis check on a specific project (by default, `project_dir` is the current working directory}:
+
+`pio check --project-dir {{project_dir}}`
```

Comment:
This could easily be split into two examples:
```suggestion
- Perform a basic analysis check on the current project:

`pio check --project-dir`

- Perform a basic analysis check on a specific project:

`pio check --project-dir {{project_dir}}`
```
(Also, your closing parenthesis is a curly brace `}` 🙂)

Reply:
Why leave the `--project-dir` in the first example?

Reply:
Updated 😆